internal/pkg/service/objectstorage/cloud/azure/tags.go

```diff
@@ -0,0 +1,31 @@
+package azure
+
+import (
+	"strings"
+
+	"github.com/giantswarm/object-storage-operator/api/v1alpha1"
+)
+
+func sanitizeTagKey(tagName string) string {
+	return strings.ReplaceAll(tagName, "-", "_")
+}
+
+func (s AzureObjectStorageAdapter) getBucketTags(bucket *v1alpha1.Bucket) map[string]*string {
+	tags := make(map[string]*string)
+	for _, t := range bucket.Spec.Tags {
+		// We use this to avoid pointer issues in range loops.
+		tag := t
```

Comment:
What is this pointer issue in range loops about ?

Reply:
```suggestion
		// We use this to well known pointer issues in range loops https://stackoverflow.com/questions/45967305/copying-the-address-of-a-loop-variable-in-go
		tag := t
```

Reply:
This is a specific case when using the address of the variable declared in the for directive, but AFAIK you are only using t (or tag) here an not its address so we are fine, this is no longer happening in latest Go version so we do not need this here.

Reply:
You're right that this was fixed in 1.22

Reply:
fixed